fadingpromises/setup-fadingpromises.tp2

```diff
@@ -45,6 +46,7 @@ LANGUAGE ~Deutsch~  ~german~  ~fadingpromises/german/setup-fadingpromises.tra~
 LANGUAGE ~Italiano~ ~italian~  ~fadingpromises/italian/setup-fadingpromises.tra~
 LANGUAGE ~Russian~ ~russian~  ~fadingpromises/russian/setup-fadingpromises.tra~
 LANGUAGE ~Francais~ ~french~  ~fadingpromises/french/setup-fadingpromises.tra~
+LANGUAGE ~Simplified Chinese By Archenemy~ ~schinese~  ~fadingpromises/schinese/install.tra~
```

Comment:
Please load `setup-fadingpromises.tra` first and then `install.tra` in initial list to load items translations for oBG2